codewars12.py

```diff
@@ -0,0 +1,10 @@
+# https://www.codewars.com/kata/55b42574ff091733d900002f/train/python
+
+def friend(x):
+    true_friends = []
+    for name in x:
+        if len(name) == 4:
+            true_friends.append(name)
+    return true_friends
+
+print(friend(["Jimm", "Cari", "aret", "truehdnviegkwgvke", "sixtyiscooooool"]))
```

Comment:
Very well done here as well. If you want to be fancy, you can also use the one-liner to solve this exercise, similarly to what you have done in exercise 11: `true_friends = [name for name in x if len(name) == 4]` No need to change it though, just as info for the future. 

One of my ex teachers once said "First do it correctly, then do it correctly and fast" ;)